wp-youtube-lyte.php

```diff
@@ -840,6 +846,19 @@ function lytecache_doublecheck_activator() {
     }
 }
 
+/* this function will alter the HTML in order to utilize an existing external lazy loader, if available */
+function lyte_try_set_bgimg($html) {
```

Comment:
not very useful I think (see earlier) so to be removed?

Reply:
Removed in latest commit.